app/controllers/pet_applications_controller.rb

```diff
@@ -0,0 +1,11 @@
+class PetApplicationsController < ApplicationController
+  def create
+    pet_adopt = PetApplication.new(application_id: params[:application_id], pet_id: params[:pet_id])
+    if pet_adopt.save
+      redirect_to "/applications/#{params[:application_id]}"
+    # else
```

Comment:
Nice work putting this in a Joins controller! That's the most RESTful option IMO. I'm curious what happened with your error handling here because it's on the right path. I suppose maybe it was for test coverage purposes?